.graphqlconfig

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Subsquid GraphQL Schema",
   "schemaPath": [
-    "https://origin.squids.live/origin-squid/graphql",
+    "https://squid.subsquid.io/origin-squid/graphql",
```

Comment:
Squid changed their URL scheme a little while back - https://origin.squids.live/origin-squid/graphql is the new official URL for prod. The other still works.

Reply:
ok, I'll update!